Formula/pyside.rb

```diff
@@ -23,11 +23,18 @@ class Pyside < Formula
 
   def install
     ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
+    if MacOS.version == :big_sur
+      # Sysconfig promotes '11' to an integer which confuses the build system.
+      inreplace "build_scripts/wheel_utils.py",
```

Comment:
Has this bug been reported upstream?

Reply:
Yes, all of the details are in the commit message. There is both a big report and a patch in the review system.

Reply:
Ah. Unfortunately, commit messages are easily missed. Please document the information you provided in the commit message in a comment here.

The most important piece of information is when we can expect this workaround to be removed.